tox.ini

```diff
@@ -257,9 +252,25 @@ commands =
     pytest {toxinidir}/certbot-ci/certbot_integration_tests \
         --acme-server={env:ACME_SERVER:pebble} \
         --cov=acme --cov=certbot --cov=certbot_nginx --cov-report= \
-        --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc \
-        -W 'ignore:Unverified HTTPS request'
-    coverage report --include 'certbot/*' --show-missing --fail-under=64
+        --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc
+    coverage report --include 'certbot/*' --show-missing --fail-under=67
     coverage report --include 'certbot-nginx/*' --show-missing --fail-under=74
-passenv =
-    DOCKER_*
+passenv = DOCKER_*
+
+[testenv:integration-certbot-oldest]
+commands =
+    {[base]pip_install} .
+    {[base]pip_install} certbot-ci
+    pytest {toxinidir}/certbot-ci/certbot_integration_tests \
+        --acme-server={env:ACME_SERVER:pebble} --campaign=certbot -p no:sugar
```

Comment:
Why no sugar?

Reply:
Because in the context of oldest tests, `pytest-sugar` does not work with the pinned old versions of pytest component installed by `pip_install.py`